api/schemas/collection-item.json

```diff
@@ -0,0 +1,25 @@
+{
+    "type": "object",
+    "description": "Describes how a Flow is collected into another (e.g. multi-essence) Flow",
+    "title": "Collection Item",
+    "required": [
+        "id",
+        "slot",
+        "role"
+    ],
+    "properties": {
+        "id": {
+            "description": "Source or Flow Identifier of the member of this collection",
```

Comment:
Is the member required to be registered in the TAMS?

Reply:
Yes.  I'll make that clear in the spec.